pkg/injection/codemodule/installer/symlink/symlink.go

```diff
@@ -74,7 +74,7 @@ func Remove(fs afero.Fs, symlinkPath string) error {
 	if info, _ := fs.Stat(symlinkPath); info != nil {
 		log.Info("symlink to codemodule directory exists, removing it due to the possibility of the agent being installed again")
 
-		if err := fs.RemoveAll(symlinkPath); err != nil {
+		if err := fs.Remove(symlinkPath); err != nil {
```

Comment:
Why the change here? Isn't `RemoveAll` better because of potential contents of the directory?

Reply:
Do not remove what the symlink is pointing to, that is for the GC to do, you have to be sure that nothing is using it

Reply:
Be aware that `RemoveAll` would not have failed if the file does not exist, `Remove` does - As we check in the if above, it should be fine 👍